xroad-catalog-collector/src/test/java/fi/vrk/xroad/catalog/collector/mock/MockMetaServicesImpl.java

```diff
@@ -86,21 +98,29 @@ private XRoadServiceIdentifierType generateService(String serviceCode,
         return service;
     }
 
-    private static String WSDL_TEMPLATE = "<wsdl:definitions xmlns:soap=\"http://schemas.xmlsoap.org/wsdl/soap/\"\n" +
+    public static String getWSDLForService(String serviceCode, String serviceVersion) {
+        return MessageFormat.format(WSDL_TEMPLATE, serviceCode, serviceVersion);
+    }
+
+    private static final String WSDL_TEMPLATE = "<wsdl:definitions xmlns:soap=\"http://schemas.xmlsoap.org/wsdl/soap/\"\n"
```

Comment:
Maybe it would be better to store this kind of long strings in separate text files and load them from there? Could be considered in a separate task.

Reply:
https://nordic-institute.atlassian.net/browse/XRDCAT-20